frontend/src/app/shared/utils.ts

```diff
@@ -108,77 +109,40 @@ export function getK8sObjectStatus(obj: K8sObject): [string, string] {
 
 // functions for processing the InferenceService spec
 export function getPredictorType(predictor: PredictorSpec): string {
```

Comment:
Let's edit the return type to be `PredictoType` here as well, to be more explicit